src/Microsoft.PowerApps.TestEngine/PowerFx/PowerFxHelper.cs

```diff
@@ -17,15 +18,16 @@ public static class PowerFxHelper
         /// </summary>
         /// <param name="engine">Instance of an engine configured with the desired locale</param>
         /// <param name="result">Check result instance created after processing the expression</param>
+        /// <param name="culture">The locale to be used when excecuting tests</param>
         /// <returns>An enumerable of formulas extracted from the expression that are separated by chaining operator</returns>
-        public static IEnumerable<string> ExtractFormulasSeparatedByChainingOperator(Engine engine, CheckResult result)
+        public static IEnumerable<string> ExtractFormulasSeparatedByChainingOperator(Engine engine, CheckResult result, CultureInfo culture = null)
```

Comment:
What happens if we pass null? Wondering if it would be a good idea to force the caller to pass in the `culture` here..